pymc/distributions/continuous.py

```diff
@@ -524,12 +525,14 @@ class Normal(Continuous):
 
     Parameters
     ----------
-    mu: float
+    mu : tensor_like of float, default 0
         Mean.
-    sigma: float
+    sigma : tensor_like of float, optional
         Standard deviation (sigma > 0) (only required if tau is not specified).
-    tau: float
+        Defaults to 1 if neither sigma nor tau is specified.
+    tau : tensor_like of float, optional
         Precision (tau > 0) (only required if sigma is not specified).
+        Defaults to 1 if neither sigma nor tau is specified.
```

Comment:
```suggestion
```

only one of tau or sigma are used, I think it's better to leave the explanation as no value for tau or sigma -> sigma=1 (tau=1 is equivalent to sigma=1 but that might still be confusing for many pymc users)